drms/version.py

```diff
@@ -0,0 +1,40 @@
+# NOTE: First try _dev.scm_version if it exists and setuptools_scm is installed
```

Comment:
We should add this to the package template as well as core

Reply:
Yes we should.